File: 240327/softeer-6283.cpp
#include<iostream>
#include<vector>

using namespace std;

int main(int argc, char** argv)
{
    vector<int> gear(8);
    for(int i=0; i<8; i++){
        int number;
        cin >> number;
        gear[i] = number;
    }

    bool asc = true;
    bool des = true;

    for(int i=0; i<7; i++){
        if(gear[i+1] - gear[i] != 1){
            asc = false;
        }
        if(gear[i+1] - gear[i] != -1){
            des = false;
        }
    }

    if(asc == true){
        cout << "ascending";
    }
    if(des == true){
        cout << "descending";
    }
    if(asc == des){
        cout << "mixed";
    }
   return 0;
}

File: 240327/softeer-6283.py
gear = [int(num) for num in input().split()]

asc = True
des = True
for index in range(7):
    if gear[index+1] - gear[index] != 1:
        asc = False

    if gear[index+1] - gear[index] != -1:
        des = False

if asc == True:
    print('ascending')
if des == True:
    print('descending')
if asc == des:
    print('mixed')